src/hooks/useNotificationTypes.tsx

```diff
@@ -232,6 +234,11 @@ export const notificationTypes: NotificationTypeConfig[] = [
             },
           });
 
+          track(AnalyticsEvent.TransferNotification, {
+            type: TransferType.deposit.name,
```

Comment:
umm is it always deposit? or can we infer it from `transferType`?

Reply:
also wonder if we need more metadata here depends on why we want to track this

Reply:
also lol yeah this should have been using the type variable 😵